service/src/test/java/uk/gov/hmcts/reform/fpl/utils/PeopleInCaseHelperTest.java

```diff
@@ -11,12 +13,46 @@
 import java.util.List;
 
 import static org.assertj.core.api.Assertions.assertThat;
+import static uk.gov.hmcts.reform.fpl.utils.CaseDataGeneratorHelper.createPopulatedApplicants;
 import static uk.gov.hmcts.reform.fpl.utils.CaseDataGeneratorHelper.createRespondents;
 import static uk.gov.hmcts.reform.fpl.utils.ElementUtils.wrapElements;
 import static uk.gov.hmcts.reform.fpl.utils.PeopleInCaseHelper.formatRepresentativesForPostNotification;
+import static uk.gov.hmcts.reform.fpl.utils.PeopleInCaseHelper.getFirstApplicantName;
 import static uk.gov.hmcts.reform.fpl.utils.PeopleInCaseHelper.getFirstRespondentLastName;
 
 class PeopleInCaseHelperTest {
+
+    @Test
+    void shouldReturnFirstApplicantNameWhenFirstApplicantWithNamePresent() {
+
```

Comment:
Don't need the blank lines at the start of this and the 2 tests below

Reply:
done